src/Goerli-DssSpell.t.base.sol

```diff
@@ -2308,8 +2325,7 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
         hevm.warp(block.timestamp + lerp.duration() / 2);
         assertEq(getMat(_ilk), _startMat * RAY / 100);
         lerp.tick();
-        assertGt(getMat(_ilk), _lowMidMat * RAY / 100);
-        assertLt(getMat(_ilk), _highMidMat * RAY / 100);
+        assertEqApprox(getMat(_ilk), _midMat * RAY / 100, _gapMidMat * RAY / 100);
```

Comment:
Why do you really need this midMat and gapMidMat?
I think you can just make the (start + end) / 2 and in tolerance just put a few weis do rounding.